src/Http.ts

```diff
@@ -2,6 +2,7 @@ import Axios, { AxiosInstance } from 'axios'
 
 export default class Http {
   public host: string;
+  public spreeToken: string
```

Comment:
@igorbezsmertnyi let's not use the `spreeToken` name - it will confuse people thinking it's the same as API v1 spree token, let's use `orderToken` please

Reply:
@damianlegawiec, after we add user authorization, according to your comment "`czy pozniej bedzie mozna dodac wiecej headersow? bo dla signed in users (out of scope teraz) bedzie potrzebny drugi header - Bearer token`" , `spreeToken` will be changed to the object with the fields
```
spreeToken = {
   userToken: string
  orderToken: string
}
```